src/main/scala/io/epiphanous/flinkrunner/model/sink/JdbcSinkConfig.scala

```diff
@@ -621,13 +611,9 @@ case class JdbcSinkConfig[ADT <: FlinkEvent](
         data.get(column.name) match {
           case Some(v) =>
             val value = v match {
-              case ts: Instant       => Timestamp.from(ts)
-              case Some(ts: Instant) => Timestamp.from(ts)
-              case Some(x)           => x
-              case Some(m: Map[String, Any]) => encoder.encode(m)
-              case m: Map[String, Any] => encoder.encode(m)
-              case None              => null
-              case _                 => v
+              case Some(x) => _matcher(x)
+              case x => _matcher(x)
+              case null | None    => null
```

Comment:
The first two cases (lines 614-615) are exhaustive, so line 616 will never be reached. Line 616 needs to be put first in order for all three cases to be processed properly.

Reply:
This was fixed. I just saw your review comment.